app/main/plugins.js

```diff
@@ -7,10 +7,7 @@ const { api } = require('dext-core-utils');
 const is = require('is_js');
 const MarkdownIt = require('markdown-it');
 const { PLUGIN_PATH } = require('../../utils/paths');
-const { MAX_RESULTS } = require('../constants');
-
-// @TODO: Move to a `constants` file
-const isDev = process.env.NODE_ENV === 'development';
+const { MAX_RESULTS, IS_DEV: isDev } = require('../constants');
```

Comment:
We can remove the alias here as well and just update all references below to `IS_DEV`.